### DISCUSSION TITLE
4.2.0 brightness slider not corresponding to 13" M4 iPad Pro brightness

### DISCUSSION THREAD

**Discussion by tetanus** [MonitorControl/MonitorControl#1571] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Using an 13" M4 iPad Pro 512GB with 14" M2 Pro MacBook Pro running macOS Sonoma 14.5 (23F79). When setting the brightness slider in the Sidecar Display to any value, it does not correspond to the brightness slider on the iPad Pro. 

### Steps to reproduce

1. On a Mac, set the brightness slider of the iPad Pro—as a Sidecar Display—to 100%. The iPad Pro screen will brighten.
2. Press the brightness up on a Magic Keyboard connected to the iPad Pro and you will see that the brightness slider value stays at the value previously set on the iPad.

### Expected behavior

Brightness value on MonitorControl should correspond to exactly that on the iPad Pro.

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.5 (23F79)
- Mac model: 14" MacBook Pro 2023, M2 Pro
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): MacBook Pro screen
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This works as expected. The app uses simple overly on the Sidecar screen (which is a macOS virtual screen streamed to the iPad's Sidecar app), it does not control the iPad's hardware brightness directly.
